session.go

```diff
@@ -444,7 +453,7 @@ func ParseURL(url string) (*DialInfo, error) {
 			}
 			fallthrough
 		default:
-			return nil, errors.New("unsupported connection URL option: " + opt.key + "=" + opt.value)
+			fmt.Println("WARN: unsupported connection URL option: " + opt.key + "=" + opt.value)
```

Comment:
Just curious why the change in api behavior? If an invalid operation is provided why not keep the error? Is there a client that is sending in an invalid option that can not be corrected? 
This changes the current API response so less chance of it getting into upstream. Also it seems this might hide more issues like a mistyped case in the url.

Reply:
No intention of getting this into the upstream - this behavior is more of an opinion that having an unknown query parameter in your connection string shouldn't crash or prevent the program from starting.